### ISSUE TITLE
Implement `TextUnit_Format` for ITextRangeProvider

### ISSUE THREAD

**Issue by yaakovschectman** [flutter/flutter#117792]
Recently added UIA implementation lacks the case for `TextUnit_Format` for moving text range endpoints, as our mojom enums currently lack `kFormatStart/End`. These additions will be necessary in order to support moving text selections by format.

**Comment by yaakovschectman** [comment]
Unassigning until UI is planned.
